index.js

```diff
@@ -30,7 +30,16 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  if (movies.length === 0) {
+    throw "There Are No Movies Available"
+  }
+  let titlesOfMovies = movies.map((movie) => {
```

Comment:
There's no need to save the  array that `.map` returns into an intermediate variable. You _can_, of course, and you may even ultimately prefer it, but see if you can get some practice returning the result of `.map` directly. See if you still pass the test!